godmin.gemspec

```diff
@@ -34,6 +34,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "bootsnap"
   gem.add_development_dependency "byebug"
   gem.add_development_dependency "capybara"
+  gem.add_development_dependency "cuprite"
```

Comment:
Should we also remove poltergeist from the dependencies?

Reply:
My bad, will fix.